CHANGELOG.md

```diff
@@ -25,6 +25,13 @@
 - `vscode-apollo`
   - <First `vscode-apollo` related entry goes here>
 
+## `apollo@2.28.0`
```

Comment:
It looks like the `CHANGELOG` was a bit out of date, so I moved the items that were sitting in `Upcoming` down into a section for the latest version.